src/main/java/io/dropwizard/elasticsearch/config/EsConfiguration.java

```diff
@@ -24,7 +24,7 @@
     private String clusterName = "elasticsearch";
 
     @JsonProperty
-    private boolean nodeClient = true;
+    private boolean nodeClient = false;
```

Comment:
Since the node client isn't supported anymore, this setting should be removed.

Reply:
Instead, we could add support for the Elasticsearch High Level REST Client:
* https://www.elastic.co/guide/en/elasticsearch/client/java-rest/6.x/java-rest-high.html
* https://www.elastic.co/blog/the-elasticsearch-java-high-level-rest-client-is-out